OpenRA.Mods.Common/Traits/Captures.cs

```diff
@@ -121,15 +121,15 @@ public override bool CanTargetActor(Actor self, Actor target, TargetModifiers mo
 
 			public override bool CanTargetFrozenActor(Actor self, FrozenActor target, TargetModifiers modifiers, ref string cursor)
 			{
-				var c = target.Info.TraitInfoOrDefault<CapturableInfo>();
+				var c = target.Actor.TraitOrDefault<Capturable>();
```

Comment:
I expect this will crash if the underlying actor has been killed / sold / etc.  You will need to make sure that `target.Actor` is valid to check first.